projects/driver_controls/src/analog_io.c

```diff
@@ -0,0 +1,78 @@
+#include <stddef.h>
+
+#include "analog_io.h"
+#include "adc.h"
+#include "gpio.h"
+#include "event_queue.h"
+#include "input_event.h"
+
+// Arbitrary thresholds for gas pedal
+#define COAST_THRESHOLD 1000
+#define DRIVE_THRESHOLD 3000
+
+// Analog device identifiers
+typedef enum {
+  ANALOG_GAS_PEDAL = 0,
+  NUM_ANALOG_IO_INPUTS
+} AnalogIODevices;
+
+// Lookup table for device ids based on ADC channel
+static AnalogIODevices s_analog_devices[] = {
+  [ADC_CHANNEL_11] = ANALOG_GAS_PEDAL
```

Comment:
If you're going to be hardcoding it here already might as well make it a #define :P